Makefile

```diff
@@ -0,0 +1,45 @@
+# Copyright 2016 The Kubernetes Authors. All rights reserved#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License..
+
+# Procedure for change:
+# 1. Bump the tag number.
+# 2. Create an image for your GCR private bucket or Docker Hub bucket and push there: make build push
+# 3. Temporarily adjust any config files etc. that reference gcr.io/google_containers/cluster-proportional-autoscaler:$(TAG)
+#    to use your chosen private bucket instead.
+# 4. Test etc.
+# 5. Issue PR.
+# 6. When PR is approved make the gcr.io version of the image: make build push
+# 7. Revert the references to kubernetes/addon-resizer:$(TAG) to gcr.io/google_containers/cluster-proportional-autoscaler:$(TAG)
+
+OUT_DIR = build
```

Comment:
prefer using := to = in Makefile, unless the Makefile needs to be portable

http://stackoverflow.com/questions/448910/makefile-variable-assignment